### DISCUSSION TITLE
Using tensor cores and RT cores

### DISCUSSION THREAD

**Discussion by malikarjun** [nvpro-samples/vk_raytracing_tutorial_KHR#40] [Q&A] [answered]
This was originally asked in the NVIDIA and Vulkan forums. 
https://forums.developer.nvidia.com/t/using-tensor-cores-and-rt-cores/202559

I wanted to know if something like this is even possible before starting out.  Due to lack of responses in other forums, I am posting here.

Thanks

**Comment by pixeljetstream** [comment, marked as answer]
you can use the KHR_ray_query functionality in compute shaders and the NV_cooperative_matrix to access tensor core functionality in compute shaders as well. Here is a sample for the latter https://github.com/jeffbolznv/vk_cooperative_matrix_perf
